Sources/LocheckLogic/Expressions.swift

```diff
@@ -78,7 +78,7 @@ struct Expressions {
         pattern: Expressions.nativeArgumentExpression,
         options: [])
 
-    private static let stringsdictArgumentExpression = #"%#@(?<name>.+?)@"#
+    private static let stringsdictArgumentExpression = #"%[0-9]*\$?#@(?<name>.+?)@"#
```

Comment:
strictly speaking, shouldn't there be parentheses around `[0-9]*\$` so that the entire thing is question-marked at once or not at all?

i.e. the latter instead of the former, since the last two are spurious matches that are incorrect syntax afaik?

![Screenshot 2023-04-25 at 5 05 29 PM](https://user-images.githubusercontent.com/87383499/234433708-ab369748-0ade-4f88-8bbb-918fefde4f0c.png)

![Screenshot 2023-04-25 at 5 05 45 PM](https://user-images.githubusercontent.com/87383499/234433748-b0260b62-8bfc-4d45-b6c1-00ce6a528f72.png)